cognite/client/_api/postgres_gateway/users.py

```diff
@@ -120,7 +120,7 @@ def create(self, user: UserWrite | Sequence[UserWrite]) -> User | UserList:
     def update(self, items: UserUpdate) -> User: ...
```

Comment:
This needs to accept UserWrite aswell, I guess

Reply:
Missed that one, done.